recipes/festival/meta.yaml

```
@@ -0,0 +1,97 @@
+{% set name = "festival" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/festival/archive/5db6ee949f4e4e9da25c87bd03b90e69f9011393.tar.gz
+  sha256: 9f0d0e4f0e91fca4430ee4a6ca4d3d24bac444f36b4f4e9653c6119aadfae13e
+
+build:
+  run_exports:
+    - {{ pin_subpackage('festival', max_pin='x.x') }}
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make  # [linux]
+    - libgomp  # [linux]
+    - libestools-dev  # [linux]
+    - libestools-static  # [linux]  This is needed to satisfy Make, but not used
+
+  host:
+    - libestools  # [linux]
+  run:
+    - libestools  # [linux]
```

build is for build tools. host is for libraries which will be linked against and need to match the target platform. run is for dependencies that do not need to be around during build but are needed at runtime.

Should be something like this.

```suggestion
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - make  # [linux]
    - libgomp  # [linux]
  host:
    - libestools-dev  # [linux]
    - libestools-static  # [linux]  This is needed to satisfy Make, but not used
```

---

Should the -dev package really be in host section? I understand "host" as "build"+"run", but the -dev version is only required during build.

---

Host is not build "build + run". It is the list of packages which must match the target platform when cross-compiling. I'm not sure why it is called 'host' instead of 'target'.

https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#host

> It represents packages that need to be specific to the target platform when the target platform is not necessarily the same as the native build platform.